conf/.env.mysql

```diff
@@ -13,7 +13,7 @@ APP_SECRET=__RANDOM_KEY__
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data/kimai.sqlite"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=mysql://__DB_USER__:__DB_PASSWORD__@127.0.0.1:3306/__DB_NAME__
+DATABASE_URL=mysql://__DB_NAME__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```

Comment:
```suggestion
DATABASE_URL=mysql://__DB_USER__:__DB_PWD__@127.0.0.1:3306/__DB_NAME__
```

Reply:
Thanks for the feedback, this is not really a issue as `db_user=$db_name`

Reply:
Interesting detail about YH that I didn't know. Thanks for the explanation. Still looks like a bug though 😁

Reply:
I will use the `ynh_add_config`helper that replace those value automatically But for now you are right It is a bug. 👍